region/norfair/east.json

```diff
@@ -5213,7 +5213,7 @@
                     "h_canNavigateHeatRooms",
                     "canSuitlessLavaDive",
                     "canIframeSpikeJump",
-                    "canWalljump",
+                    "canStaggeredWalljump",
```

Comment:
HJBless dive should have `canFastWalljumpClimb` for the spike climb, and `canStaggeredWallJump` for the right side climb.
`canUseEnemies` should also be included in all of these stand on the Namihe strats too.